### DISCUSSION TITLE
Welcome to kaizoku Discussions!

### DISCUSSION THREAD

**Discussion by oae** [oae/kaizoku#24] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by eswiig** [comment]
Just discovered Kaizoku and I'm super impressed. This is almost everything I wanted in a Manga Downloader. 

The only feature I really want is to be able to customize the naming convention when it comes to the .cbz and  filenames.

Really awesome work! Please keep it up and don't let stop the updates... more people needs to know about this app.

**Reply by oae** [reply]
Thank you. Naming convention change is asked a lot. I plan to implement it but not sure when.

**Comment by alejakun** [comment]
Hi, I just found out about kaizoku, after a bumpy start I finally got it to work correctly within my environment, however I'm wondering if it would be possible to create (other than creating a new instance), a new library with it's own rules, notifications, directory, etc...

Thanks for the great work!
